packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -176,9 +176,34 @@ const BlockInspector = ( { showNoBlockSelectedMessage = true } ) => {
 	const blockInspectorAnimationSettings = useSelect(
 		( select ) => {
 			if ( blockType ) {
+				// Get the global blockInspectorAnimationSettings.
 				const globalBlockInspectorAnimationSettings =
```

Comment:
Let's drop this comment as the variable is named `globalBlockInspectorAnimationSettings`